src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.2
++++++
+* Minor fixes.
```

Comment:
Is this a minor fix or a breaking change?  Most of these changes signify breaking changes, so we can't necessarily rely on our boilerplate text here.

Reply:
Thanks